### DISCUSSION TITLE
Issues running Define Equilibration node in Pipeline

### DISCUSSION THREAD

**Discussion by cmmercer** [NMGRL/pychron#2188] [Q&A]
I have a run that had some odd behavior on several detectors for the first part of the run, and wanted to try trimming the front of the data with the Define Equilibration node in Pipeline. I got a series of different error messages as I was trying to run it; see attached log file.
[pychron.current.log](https://github.com/NMGRL/pychron/files/10187452/pychron.current.log)

**Comment by jirhiker** [comment]
The short of it is do not use Define Equilibration

Use https://github.com/NMGRL/pychron/wiki/Data-Reduction-Tips
